Containerfile

```diff
@@ -8,6 +8,8 @@ RUN apt update -y && apt install -y libgdal-dev libproj-dev libmpfr-dev && apt c
 
 # PrevMap requires Terra and raster, which fail to install inside R. So install with mamba instead:
 RUN mamba install \
+    cargo-llvm-cov \
+    clarabel \
```

Comment:
I didn't see test cases for these or mentions about them in the commit/description.  Were these added to try to fix filling and as such are no longer needed or are they intended additions?

Reply:
They were intended they caused conflicts or weren't supported anymore We just never fixed them. This image isn't for a class so it's not like there's an absolute authority for the packages. Worst case if someone wants them back we can add them, but I don't think they do.